packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -1,23 +1,36 @@
 'use client';
+
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { styled, useTheme, SxProps } from '@mui/material';
+
+// MUI: Styles & Theme
+import { styled, useTheme, SxProps } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import type {} from '@mui/material/themeCssVarsAugmentation';
+
+// MUI: Components
 import MuiAppBar from '@mui/material/AppBar';
 import Box from '@mui/material/Box';
 import Drawer from '@mui/material/Drawer';
 import IconButton from '@mui/material/IconButton';
 import Stack from '@mui/material/Stack';
 import Toolbar from '@mui/material/Toolbar';
 import Tooltip from '@mui/material/Tooltip';
-import useMediaQuery from '@mui/material/useMediaQuery';
-import type {} from '@mui/material/themeCssVarsAugmentation';
+
+// MUI: Icons
 import MenuIcon from '@mui/icons-material/Menu';
 import MenuOpenIcon from '@mui/icons-material/MenuOpen';
+
+// Toolpad: Context
 import { BrandingContext, NavigationContext, WindowContext } from '../shared/context';
+
+// Toolpad: Components
 import { Account, type AccountProps } from '../Account';
+import { AppTitle, type AppTitleProps } from './AppTitle';
 import { DashboardSidebarSubNavigation } from './DashboardSidebarSubNavigation';
 import { ToolbarActions } from './ToolbarActions';
```

Comment:
The `ToolbarActions` component would now need to include an `Account` on the right side.
As we're deprecating the `toolbarAccount` slot we can make it default to being empty, I suppose.